hcl/hcl.go

```diff
@@ -264,7 +268,7 @@ func ParseGlobalsBlocks(dir string) (HCLBlocks, error) {
 // ParseGenerateHCLBlocks parses all Terramate files on the given dir, returning
 // only generate_hcl blocks (other blocks are discarded).
 // generate_hcl blocks are validated, so the caller can expect valid blocks only or an error.
-func ParseGenerateHCLBlocks(dir string) (HCLBlocks, error) {
+func ParseGenerateHCLBlocks(dir string) (Blocks, error) {
```

Comment:
```suggestion
func ParseGenerateBlocks(dir string) (Blocks, error) {
```

Reply:
hmm in this case the block being parsed is "generate_hcl", hence ParseGenerateHCL. For generate_file it would be ParseGenerateFile. If we go just with ParseGenerate it seems vague.